packages/wxt-demo/package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
+    "eslint": "^9.5.0",
```

Comment:
Same here. Create a E2E test that uses eslint 9 instead.

Reply:
Do we even need ESLint included? I had added it for the demo stub, but since that config is being removed, I don't think we need ESLint even added to the demo package

Reply:
No, we should be good. I'll remove the types package.